kb/compiling-and-building/how-do-i-configure-mbedtls.md

```diff
@@ -22,11 +22,11 @@ The `config.py` script automatically finds the `mbedtls_config.h` file when it r
 
 ## Alternative configuration files
 
-You might want to keep the custom configuration file for your application outside the Mbed TLS source tree. You can do this by defining the macro `MBEDTLS_CONFIG_FILE` for the desired filename (including the quote or angular brackets) at compile time. For example, using **make**:
+You might want to keep the custom configuration file for your application outside the Mbed TLS source tree. You can do this by defining the macro `MBEDTLS_CONFIG_FILE` for the desired filename (including the quote or angular brackets) at compile time. For example, using **make**: (Note: The angle brackets `<>` are included.)
```

Comment:
Do we really need to say three times that the angle brackets are included???

Reply:
Ah my apologies, I misread [your previous comment](https://github.com/Mbed-TLS/mbedtls-docs/pull/94#discussion_r1177645377). However, I' prefer to keep this note and remove the original one because it's more obviously linked to the code snippet

Reply:
There is now one and only one comment about angle brackets in a more obvious place.